README.md

```diff
@@ -1,6 +1,16 @@
 # starknet-rivet
 
-> a chrome extension tools built with Vite + React, and Manifest v3
+Starknet Rivet is a developer wallet tailored specifically for the Starknet ecosystem. 
+
+It aims to streamline your development process, offering:
+
+1. Easy Onboarding: Set up a local devnet effortlessly
+2. Block and Transaction Explorer: Inspect blocks and transactions
```

Comment:
For now, you can not inspect the block (you see the block number and the number of transactions for each block but you can not see the transactions hash or anything).
But you can play with blocks, like create new blocks, abort blocks, set intervals, etc...